js/modules/k6/http/http.go

```diff
@@ -43,6 +44,9 @@ const (
 	HTTP_METHOD_OPTIONS = "OPTIONS"
 )
 
+// ErrJarForbiddenInInitContext is used when a cookie jar was made in the init context
+var ErrJarForbiddenInInitContext = errors.New("Making cookie jars in the init context is not supported")
```

Comment:
I like the distinct messages and error values, but `errors.New()` will make checking for init errors as a group (so we can use an appropriate exit code for example) very difficult. Can you create a new custom error type (`InitError` or something like that) that we can later use to group check all of those errors?